modules/products/bamboo/locals.tf

```diff
@@ -2,19 +2,20 @@ locals {
   product_name = "bamboo"
 
   helm_chart_repository     = "https://atlassian.github.io/data-center-helm-charts"
-  bamboo_helm_chart_version = "1.0.0"
-  agent_helm_chart_version  = "1.0.0"
+  bamboo_helm_chart_version = var.bamboo_configuration["helm_ver"]
```

Comment:
I think the complete form `helm_version` will be clearer.

Reply:
Thanks, is addressed.